src/test/java/uk/gov/pay/connector/paymentprocessor/service/CardAuthoriseServiceTest.java

```diff
@@ -143,6 +144,9 @@ class CardAuthoriseServiceTest extends CardServiceTest {
     private final ChargeEntity charge = createNewChargeWith(1L, ENTERING_CARD_DETAILS);
     private final String chargeExternalId = charge.getExternalId();
 
+    private final CollectorRegistry collectorRegistry = CollectorRegistry.defaultRegistry;
+    private static final String[] labelNames = new String[] {"paymentProvider", "gatewayAccountType", "billingAddressPresent", "authorisationResult"};
```

Comment:
(minor) Formatting in this file for additional spaces.
```suggestion
    private static final String[] labelNames = new String[]{"paymentProvider", "gatewayAccountType", "billingAddressPresent", "authorisationResult"};
```